src/main/java/org/fundacionjala/core/config/EnvironmentProperties.java

```diff
@@ -0,0 +1,41 @@
+package org.fundacionjala.core.config;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public final class EnvironmentProperties {
+
+    /**
+     * Constructor of the Class.
+     */
+    private EnvironmentProperties() {
+
+    }
+    /**
+     * Initialize a new GetProperties.
+     * @return GetProperties.
+     */
+    public static EnvironmentProperties getInstance() {
+        return new EnvironmentProperties();
+    }
+
+    /**
+     * Gets properties from properties file.
+     *
+     * @param path - Path of Gradle properties file.
+     * @return Properties.
+     */
+    public static Properties getProperties(final String path) {
```

Comment:
Looks like this class is a util to read properties file. It should not be singleton, and please rename the class, and move it to a utils package inside core